internal/config/identity/openid/jwt.go

```diff
@@ -181,8 +192,14 @@ func (r *Config) Validate(ctx context.Context, arn arn.ARN, token, accessToken,
 		return ErrTokenExpired
 	}
 
-	if err = updateClaimsExpiry(dsecs, claims); err != nil {
-		return err
+	timeout := env.Get(config.EnvMinioStsDuration, "")
```

Comment:
if we already are using the env variable in updateClaimsExpiry, there's no need to do the replace here.